source/06-utility/build-test/Component-jsx.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import moduleTS from './_module-ts';
+
+function ComponentJsx() {
+  return <div>{moduleTS()}</div>;
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8821041)
```suggestion
}
```